zenoh-kotlin/build.gradle.kts

```diff
@@ -208,7 +208,7 @@ fun buildZenohJNI(mode: BuildMode = BuildMode.DEBUG) {
     }
 
     val result = project.exec {
-        commandLine(*(cargoCommand.toTypedArray()), "--manifest-path", "../zenoh-jni/Cargo.toml")
+        commandLine(*(cargoCommand.toTypedArray()), "--manifest-path", "./zenoh-jni/Cargo.toml")
```

Comment:
There are a number of other places in the file that uses the relative path `../`, shouldn't those be updated as well?

Reply:
Looks like CI passes with just the one change in the relative dir. I wonder if it might cause problems during release though

Reply:
I have double checked doing publications to maven local and the native libs are loaded properly. Seems that the Gradle upgrade issue only affects here when running a command this way using commandLine (which is a trick for automatically building the native library before building the packages). These changes should work fine.